Kubernetes/legos/k8s_get_unbound_pvcs/k8s_get_unbound_pvcs.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (c) 2023 unSkript.com
+# All rights reserved.
+#
+from typing import Tuple
+from kubernetes import client
+from kubernetes.client.rest import ApiException
+from pydantic import BaseModel, Field
+
+
+class InputSchema(BaseModel):
+    namespace: str = Field (
+        '',
+        description='K8S Namespace',
+        title="K8S Namespace"
+    )
+
+def k8s_get_unbound_pvcs_printer(output):
+    if output is None:
+        return 
+    
+    print(output)
+
+def k8s_get_unbound_pvcs(handle, namespace:str = '') -> Tuple:
+    """k8s_get_unbound_pvcs This function all unbound PVCS and returns them back
+
+       :type handle: Object
+       :param handle: Object returned from the task.validate(...) function
+
+       :type namespace: str
+       :param namespace: Kubernetes Namespace 
+
+       :rtype: Tuple Result in tuple format.  
+    """
+    if handle.client_side_validation != True:
+        raise Exception(f"K8S Connector is invalid {handle}")
+
+    v1 = client.CoreV1Api(api_client=handle)
+
+    # Get all PVCs in the cluster
+    if not namespace:
```

Comment:
This is an expensive loop you have going on in here, pvc X pods. why not use something like this https://medium.com/@dinkarbhagia/cleaning-up-unused-pvc-from-k8s-cluster-using-python-k8s-sdk-b7407d881df4

Reply:
Done

Reply:
Still see the same code, what did we do here?